tests/test_integration.py

```diff
@@ -292,7 +289,13 @@ def test_list_relationship(seed_dynamic_data):
         ("required-all-components.json", "required.json", 8, 27),
         ("favourite-colours-sarah.json", "colours.json", 4, 1),
         ("funding-required-cof-25.json", "funding-required.json", 12, 21),
-        ("Organisation-and-local-authority-information-template.json", "local-authority-and-other-organisation-information.json",16, 24),  # noqa: E501
+        (
+            "Organisation-and-local-authority-information-template.json",
+            "local-authority-and-other-organisation-information.json",
+            16,
+            24,
+        ),  # noqa: E501
+        ("test-section.json", "section.json", 3, 1),
```

Comment:
Do we have any checks in this test that confirm the sections are added correctly?